master/buildbot/changes/gerritchangesource.py

```diff
@@ -300,12 +300,12 @@ def describe(self):
             project = "all projects"
 
         if self.branch is not None:
-           branch = "branch %s" % self.branch
+           branch = "branch '%s'" % self.branch
         else:
            branch = "all branches"
 
         if not self.process:
             status = "[NOT CONNECTED - check log]"
-        msg = ("GerritChangeSource watching the remote "
+        msg = ("GerritChangeSource watching project %s (%s) in remote "
```

Comment:
Corrected printing "project" twice. Also brought the project string in line with the branch string. Decoration is still part of the branch and project strings because phrasing varies based on if they are defined or not.